### ISSUE TITLE
Tumor microbiome diversity influences papillary thyroid cancer invasion

### ISSUE THREAD

**Issue by Rahila-me** [waldronlab/BugSigDBcuration#377]
Tumor microbiome diversity influences papillary thyroid cancer invasion - Lijuan Yuan - nature
https://www.nature.com/articles/s42003-022-03814-x

**Comment by AleruDivine** [comment]
Good morning @SvetlanaUP, I'd like to be assigned this paper for curation.
Thank you so much!

**Comment by AleruDivine** [comment]
Thank you so much @SvetlanaUP!

**Comment by AleruDivine** [comment]
Hello @cmirzayi, I'd like to put this study up for review.
Thank you so much!
Link: https://bugsigdb.org/Study_1033

**Comment by vandana123maddi** [comment]
Hello @cmirzayi and @SvetlanaUP  Please I would love to be assigned to this 
Thank you. 🤗

**Comment by SvetlanaUP** [comment]
Hi @vandana123maddi, this paper has been already curated and currently is under review (see label: "under review"). Please select other "paper to curate".

**Comment by KateRasheed** [comment]
Good morning @SvetlanaUP . I'll like to be assigned this paper please.

**Comment by KateRasheed** [comment]
Good evening @AleruDivine . Welldone on this curation. Here are some observations on the study:

- For experiment 1, the data transformation is relative abundance and MHT is no.
- Please change hyphomicrobiales to unclassified hyphomicrobiales
- Also change Welch test data transformation to relative abundance.
- Please request that experiment 2, 7 be deleted since there was no significant taxa.
- For experiment 8, there was no MHT correction done.
- Experiment 9-17, 18-21 should be deleted as there was no comparison done. The only comparison with significant taxa is in Fig. 3b (Anti-TG and Anti-TPO). The significant taxa is Akkermansia.
- The arscine square root transformation I believe was applied to Spearman correlation.
- There is an additional experiment in Supplementary Figure 2A done with Wilcoxon. There are also additional experiments in Supplementary Data 4 for Kruskal-Wallis test (Sheet 2)
- For experiment 18, the analysis was between FT3 and T3

Thank you so much.

**Comment by AleruDivine** [comment]
Thank you for the review @KateRasheed. 
I’ll go through your points and make corrections.

**Comment by AleruDivine** [comment]
Hi @KateRasheed I have made all corrections.

For Figure S2A, following the curation policy, we use the test that reports a larger number of signatures whenever signatures overlap.  In this case, it was figure 2h. I have left a note acknowledging Supplementary Figure 2A and the statistical test used.

I have added the experiments from supplementary data 4. I edited both experiments 2 and 9 so no need to delete them. 

For experiments 9 - 21, I believe they should be curated, for example, Prevotella was decreased for high levels of thyroid-stimulating hormone (TSH) relative to low levels of TSH. I have added group 0 names for each of them as these were omitted before. Please check this office hours [recording](https://community-bioc.slack.com/archives/C04RATV9VCY/p1734017777427699) for clarity. Time-stamp: 34:00 

FT3 and T3 are both stand-alone experiments (high levels vs low levels).

Thank you so much!

**Comment by AleruDivine** [comment]
@SvetlanaUP Please delete signatures 3 and 4 in experiment 24.
Thank you so much!

**Comment by KateRasheed** [comment]
Good morning @AleruDivine. Here are some observations on the study

- For experiment 2, 7, 22 the signatures abundance direction are inverted. Signature 2 is supposed to be increased. I think the confusion is from the ordering of the column; T1_sd is supposed to be T1_mean. The p-value column should be for the statistics column. 
- Also, for signature 2 in experiment 2, Butyricoccaceae is Butyricoccaceae UCG-009 while Shewella should be replaced with Truepera - ID 332248.
- For experiment 2 signature 1, Comamonadaceae should be unclassified Comamonadaceae and Shewanella, Pyramidobacter, Streptobacillus, Prevotellaceae_UCG-004, Spirochaeta, Crossiella, Fournierella, Desulfallas-Sporotomaculum, Ruegeria, URHD0088, NB1-j, Facklamia, Eubacterium, Proteus, Sandaracinobacter, Sulfurisoma are missing
- For experiment 7 signature 2, Prevotellaceae_UCG-004 should be curated as it is. I can't find where it's resolved in the discussion page. Proteus is missing while in signature 1, Spirochaeta, Desulfallas-Sporotomaculum, Crossiela, Truepera, URHD0088, Facklamia, Sulfurisoma are missing
- For experiment 9 signature 1, Clostridia should begin with capital letter
- For experiment 13 signature 1, Chloroplast is missing
- For experiment 14 signature 1, Burkholderia-Caballeronia-Paraburkholderia should be curated together not separated; they are related species.
- For experiment 22 signatures are mixed-up: Granulicatella, Candidatus Gastranaerophilales, Monoglobus, Halomonas, Streptococcus, Haemophilus are increased while Oscillibacter, Chitinophagaceae are decreased. Please check again.

I'm yet to review 23-25. Kindly make these corrections on the signature abundance direction so I can continue. Thank you so much.

**Comment by AleruDivine** [comment]
You are absolutely right. I used the SD columns because they’re the mean values. I wrote these signatures before curating and I mixed the abundance directions up. That has been corrected now. 

For Prevotella UCG 004, correct me if I’m wrong, but I believe the explanation by Levi covers all UCGs to be curated as uncultured bacterium. 

I will confirm the missing signatures and correct accordingly. 

Thank you @KateRasheed

**Comment by AleruDivine** [comment]
All corrections have been made @KateRasheed. 
Waiting on the rest of the review. Thanks!

**Comment by SvetlanaUP** [comment]
Study_1033/Experiment_24/Signature 3 and 4 deleted.

**Comment by AleruDivine** [comment]
Good morning @KateRasheed, all corrections have been made to this study.
Thank you!

**Comment by KateRasheed** [comment]
Good morning @AleruDivine . Thanks for effecting most of the corrections. 

For experiment 24, "Sutterella, Methylobacterium-Methylorubrum, Ruegeria, NB1-j, Proteus is missing in signature 1" - This was not effected. They are showing on the increased signature not decreased (signature 1)

Thank you so much.

**Comment by AleruDivine** [comment]
All changes have been made @KateRasheed

**Comment by KateRasheed** [comment]
Thank you so much @AleruDivine . This review is complete @SvetlanaUP

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/Study_1033 reviewed.
